scripts/travisTest.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euxo pipefail
+
+##############################################################################
+##
+##  Travis CI test script
+##
+##############################################################################
+
+mvn -q clean package liberty:create liberty:install-feature liberty:deploy
+mvn liberty:start
+mvn failsafe:integration-test liberty:stop
+mvn failsafe:verify
```

Comment:
These seem like the basic set of tests need to be run for a guide (similar to previously running `mvn clean install`. Suggest putting this in the template as well when ready.